ethereum/contracts/relayer/coreRelayer/CoreRelayerMessages.sol

```diff
@@ -344,9 +344,6 @@ contract CoreRelayerMessages is CoreRelayerGetters {
         bool roundUp,
         IRelayProvider provider
     ) internal view returns (uint256 targetAmount) {
-        if (!provider.isChainSupported(targetChain)) {
```

Comment:
Are we making the isChainSupported check implicit now? Basically if the src and dst native currency price !== 0 implies it is supported?

Reply:
Oh no, I just moved it to the beginning of 'send/forward/resend'